ns/executor.go

```diff
@@ -59,24 +59,29 @@ func (nsexec *Executor) prepareCommandArgs(binary string, args []string) []strin
 			cmdArgs = append(cmdArgs, "--net="+nsPath)
 		}
 	}
+	if len(envs) > 0 {
+		cmdArgs = append(cmdArgs, "env", "-i")
```

Comment:
Why having "`env` `-i`"? instead of using `-e` or `--env` for each environment variable pair? 

Something like below instead?

```go
for _, env := range envs {
    cmdArgs = append(cmdArgs, "--env="+env)
}

```

Reply:
`env -i` is to executes the specified command with an empty environment and make sure the execution is not interfered with other environment variables.

Ref: https://man7.org/linux/man-pages/man1/env.1.html